src/view/com/composer/Composer.tsx

```diff
@@ -333,25 +328,6 @@ export const ComposePost = ({
     }
   }, [onPressCancel, closeAllDialogs, closeAllModals])
 
-  const {needsEmailVerification} = useEmail()
-  const emailDialogControl = useEmailDialogControl()
-
-  useEffect(() => {
-    if (needsEmailVerification) {
-      emailDialogControl.open({
```

Comment:
Like the `Conversation` view, this will cause an infinite render loop. All access to the composer is now gated, so we don't need anything like this here. For the Conversation, access is more difficult to gate, so I left that use case.